source/Engine.cpp

```diff
@@ -199,6 +199,87 @@ namespace {
 		}
 	}
 
+	const Color &GetTargetOutlineColor(int type)
+	{
+		const Color &player = *GameData::Colors().Get("ship target outline player");
```

Comment:
If those variables are only used once, why have them in the first place?

Reply:
Puts them all together, looks nicer.
I can move them into the if statement if you think that'd be better.

Reply:
dewit